observability/otel/otel-collector/otel-collector-receivers.adoc

```diff
@@ -352,6 +352,29 @@ include::snippets/technology-preview.adoc[]
 <3> The lnterval for scraping the metrics data. Accepts time units. The default value is `1m`.
 <4> The targets at which the metrics are exposed. This example scrapes the metrics from a `my-app` application in the `example` project.
 
+[id="otlpjsonfile-receiver_{context}"]
+== OTLP JSON File Receiver
+
+The OTLP JSON File Receiver extracts pipeline information from files containing data in the link:https://protobuf.dev/programming-guides/json/[ProtoJSON] format and conforming to the link:https://opentelemetry.io/docs/specs/otel/protocol/[OpenTelemetry Protocol] specification. This receiver watches a specified directory for changes: when a file is created or modified, this receiver reprocesses the complete file.
```

Comment:
TY. I reworded this sentence differently: "The receiver watches a specified directory for changes so that when a file is created or modified, the receiver reprocesses the complete file."

Reply:
TY. I changed this sentence to as follows:
"The receiver watches a specified directory for changes such as created or modified files to process."